Modules/selectmodule.c

```diff
@@ -392,33 +447,31 @@ ushort_converter(PyObject *obj, void *ptr)
     return 1;
 }
 
-PyDoc_STRVAR(poll_register_doc,
-"register(fd [, eventmask] ) -> None\n\n\
-Register a file descriptor with the polling object.\n\
-fd -- either an integer, or an object with a fileno() method returning an\n\
-      int.\n\
-events -- an optional bitmask describing the type of events to check for");
+/*[clinic input]
+select.poll.register
```

Comment:
Please use "-> NoneType". Even if currently it seems like it has no effect, it will be used in signature once https://bugs.python.org/issue31939 will be implemented.

Reply:
That syntax is for defining AC return converters, not return type annotations, so it could have unwanted effects.

Specifically, the [Argument Clinic How-To](https://docs.python.org/3/howto/clinic.html) currently states:
> (There’s also an experimental NoneType converter, which lets you return Py_None on success or NULL on failure, without having to increment the reference count on Py_None. I’m not sure it adds enough clarity to be worth using.)